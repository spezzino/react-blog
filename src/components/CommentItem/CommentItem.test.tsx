import React from "react";
import { CommentItem } from "./";
import renderer from "react-test-renderer";

test("List gets populated with items", () => {
  const component = renderer.create(
    <CommentItem
      postId={1}
      id={1}
      name="Name"
      email="email@sample.com"
      body="body"
    />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
