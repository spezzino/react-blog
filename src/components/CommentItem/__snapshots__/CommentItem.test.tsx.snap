// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List gets populated with items 1`] = `
<div
  className="MuiPaper-root MuiCard-root comment-item MuiPaper-elevation1 MuiPaper-rounded"
>
  <div
    className="MuiCardContent-root"
  >
    <h4
      className="MuiTypography-root MuiTypography-h6"
    >
      Name
    </h4>
    <p
      className="MuiTypography-root MuiTypography-body2"
    >
      body
    </p>
  </div>
</div>
`;
